app/assets/stylesheets/main.css

```diff
@@ -0,0 +1,125 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato&family=Poppins&family=Roboto:wght@400;700&family=Square+Peg&display=swap");
```

Comment:
- [ ] The details view does not look like the given design:

Your implementation:

![image](https://github.com/Benawi/Doctor_Appointment-Booking_App/assets/57421823/4c369df8-3203-4369-80d6-2f08bbc7955b)

Given design:

![image](https://github.com/Benawi/Doctor_Appointment-Booking_App/assets/57421823/1b35ef0e-c908-4299-b433-0fc41fe96e00)

Kindly, make it so your styling implementation looks more like the required design. 👍